label_studio/data_manager/actions/basic.py

```diff
@@ -54,8 +53,13 @@ def delete_tasks(project, queryset, **kwargs):
     # delete only specific tasks
     else:
         # this signal re-save the task back
-        with temporary_disconnect_signal(signals.post_delete, update_is_labeled_after_removing_annotation, Annotation):
+        with temporary_disconnect_all_signals():
             queryset.delete()
+        project.update_tasks_states(
```

Comment:
that will not recompute project.summary counters

Reply:
Fixed